tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -307,45 +307,54 @@ IncludeByKey.PASSTHROUGH {
 						}
 						Object.Widget.module-copier.1 {
 							stream_name 'Gain Capture 19'
+
 							num_input_audio_formats 2
-							num_output_audio_formats 2
-							Object.Base.audio_format.1 {
+							Object.Base.input_audio_format.1 {
 								in_bit_depth		32
 								in_valid_bit_depth	32
-								out_bit_depth		32
-								out_valid_bit_depth	32
 							}
-							Object.Base.audio_format.2 {
+							Object.Base.input_audio_format.2 {
```

Comment:
@RanderWang can you please avoid the .1/.2 and use array instead?

Reply:
sure, thanks